### DISCUSSION TITLE
how to copy a specific role dashboard widget to another role .

### DISCUSSION THREAD

**Discussion by kassimg12** [YetiForceCompany/YetiForceCRM#16022] [How to]
Hello,

I have different roles in the system. based on the hierarchical system a user can get subordinate records. therefore I wanted to have similar roles but different structures for each branch of our company. when doing so, the profile works fine but I faced mandatory dashboard creating issues for each role.

I can create a dashboard for each user in setting>standard modules>Dashboard widgets. however when I couldnt find a way to create similar dashboards by copying once dashboard for another.  is that possible?

Thank you in advance.

**Comment by vovpff** [comment]
There is no ability to duplicate dashboatd content from frontend.

You can try to duplicate rows in database and replace role id but I'm not sure what this will work well

**Comment by kassimg12** [comment]
Great hint.

**Comment by vovpff** [comment]
It works? I never tried this before
